### PR TITLE
The End Affector subsystem and its methods

### PR DESCRIPTION
**Pull request by MidgleyM**

### CODE REVIEW

File: robotbase/src/main/java/frc/robot/subsystems/EndAffectorSubsystem.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.motorcontrol.Talon;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.CAN_ID;
+import frc.robot.Constants.END_AFFECTOR;
+
+public class EndAffectorSubsystem extends SubsystemBase {
+
+  private Talon m_motor;
```
Comment (NolanCampbell2357) [inline robotbase/src/main/java/frc/robot/subsystems/EndAffectorSubsystem.java:10]: I do not know what the wpilib's talon is. Change this to using a sparkmax.

**Review by NolanCampbell2357 — CHANGES_REQUESTED** [review]
Go ahead and add the axis commands along with the requested changes

File: robotbase/src/main/java/frc/robot/subsystems/EndAffectorSubsystem.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    m_motor.setSmartCurrentLimit(END_AFFECTOR.MOTOR_STALL_LIMIT_AMPS, END_AFFECTOR.MOTOR_FREE_LIMIT_AMPS);
+    m_motor.enableVoltageCompensation(12);
+  }
+
+  public void setPivotAxisSpeed(double axisSpeed) {
+    double motorSpeed = (-axisSpeed) * END_AFFECTOR.AXIS_MAX_SPEED;
+    m_motor.set(motorSpeed);
+  }
+
+  public void setSpeed(double speedPercentage) {
```
Comment (NolanCampbell2357) [inline robotbase/src/main/java/frc/robot/subsystems/EndAffectorSubsystem.java:32]: Let's rename this to setAxisSpeed to be consistent with other subsystems.

**Review by NolanCampbell2357 — CHANGES_REQUESTED** [review]
Almost good, one rename + linting to fix

File: robotbase/src/main/java/frc/robot/subsystems/EndAffectorSubsystem.java (hunk 2 above)
Comment (MidgleyM) [reply robotbase/src/main/java/frc/robot/subsystems/EndAffectorSubsystem.java:32]: yeah i forgot to rename that. woops.

**Pull request merged**